src/components/Navbar/Navbar.jsx

```diff
@@ -1,16 +1,38 @@
-import { React } from 'react';
+import { useState, React } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { HStack, Flex, Button, Image, useMediaQuery, useToast } from '@chakra-ui/react';
+import {
+  HStack,
+  Flex,
+  Button,
+  Avatar,
+  Image,
+  Menu,
+  MenuButton,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'MenuButton' is defined but never used.